README.md

```diff
@@ -1,12 +1,8 @@
 # GOV.UK country and territory autocomplete - what it is and how to use it
 
----
-
-**This component is not currently maintained and we are unable to respond to issues and pull requests.**
+The country and territory autocomplete is a component that helps users choose countries and territories in answers. You should only use this component for services where users have to provide their country or territory.
 
-You can still raise issues or pull requests if you want to. As soon as we’re able to, we’ll prioritise dealing with security updates and any bugs that have been raised by people in the UK public sector. We cannot prioritise adding new features.
-
----
+For other types of service, you should use the accessible autocomplete. To find information about this component, go to the [accessible-autocomplete README](https://github.com/alphagov/accessible-autocomplete/blob/main/README.md).
 
 The GOV.UK country and territory autocomplete  is a simple UI widget that allows end users to find and select a location. It includes auto-complete functionality to make it faster and easier for users to find a location. For example, to select their country of birth or their current location.
```

Comment:
Is this content worth somehow combining with the first sentence on this page? (Instead of having 2 similar pieces of content separated by line 5's content re the accessible-autocomplete.)

Reply:
Perhaps move the 2nd sentence up after the 1st paragraph? 'Auto-complete' might not be needed (and spelled inconsistently) there.

Reply:
That sounds like a good idea - thanks. Will add in this change.

Reply:
Thanks - done!

Reply:
`For other types of question` maybe? the type of service doesn't have a bearing on whether you'd use this component.